communications/slack-culture.rst

```diff
@@ -83,6 +83,31 @@ You can save space, time, and distractions to others by using commonly understoo
 
 -  ``:thank_you:`` (🙏) or ``:thankyou:``
 
+.. _slack-direct-messages:
+
+Direct messages
+---------------
+
+Please keep direct messaging to a user or a set of users to minimum.
+Direct messages should be avoided for any technical discussions or asking for support, even if you are sure that you have involved all those who are interested or who can contribute.
+Instead, such discussions should be in a thread in an appropriate :ref:`discussion channel<slack-discussion-channels>`.
+Direct messages shoul be reserved for conversations on topics such as code review requests, logistics, or on other topics not directly related to work.
```

Comment:
"shoul" → "should".  The phrasing here is not great ("topics" twice, no "on" with logistics, "reserved" but a non-exhaustive list).  It might be better to be more explicit with reasons and considerations (pulling them out of the Community post) rather than trying to be prescriptive.  And the fact is that there are many, many direct messages anyway, so if we are describing the existing culture, this seems inaccurate.  (I think it is OK to try to gently nudge the culture in a different direction, but it's more difficult to overturn it, and that should probably be via RFC as well.)

Reply:
I thought the point of the dev guide is not to describe what the current culture is, but rather serve as a guide to what the culture ought to be. If we, as a team, indeed want to keep DMs to a minimum, then whenever we find ourselves in DMs that are getting technical, we can point to the dev guide and ask to move the conversation in an appropriate channel.

Reply:
Are direct messages happening a lot within the data management team? The [analytics](https://lsstc.slack.com/stats) that Eric had posted on the community post includes Science Collaborations and other parts of the project. I personally don't use a lot of direct messaging, and my impression was that data management in general uses less direct messages than other users in the workspace.